### PR TITLE
Lint warning fixes for 34, 37, 38, 39, 40

### PR DESCRIPTION
**Pull request by cotikor**

### CODE REVIEW

**Review by takeutak — APPROVED** [review]

**Comment by petermetz** [comment]
@cotikor Which issue is this for? (I assume it's one of the good first issue labeled ones we prepared for GHC OSD, right?)

File: packages/cactus-cmd-api-server/src/main/typescript/config/config-service.ts [hunk 1]
```diff
@@ -58,6 +58,7 @@ export interface ICactusApiServerOptions {
   keyPairPem: string;
   keychainSuffixKeyPairPem: string;
   minNodeVersion: string;
+  default: null
```
Comment (petermetz) [inline packages/cactus-cmd-api-server/src/main/typescript/config/config-service.ts:61]: @cotikor What is the purpose of this new property on this type?

**Review by petermetz — CHANGES_REQUESTED** [review]
@cotikor Please fix the DCO warning: there's a guide on how to do it if you click 'details' next to the failing check.
Also: Please rebase  + squash onto the latest upstream main to resolve the conflicts, let me know if you need any help with git mechanics!

**Comment by petermetz** [comment]
Closing due to inactivity, feel free to reopen as needed.

**Pull request closed without merging**